packages/lending/src/protocols/morphoblue/lending-protocol.ts

```diff
@@ -217,6 +202,27 @@ export class LendingProtocol extends Protocol {
     return false;
   }
 
+  async getBorrowAPY(marketId: string, market: MarketStruct) {
+    const { loanToken, collateralToken, oracle, irm, lltv } = getMarket(this.chainId, marketId);
+
+    const marketParams: MarketParamsStruct = {
+      loanToken: loanToken.address,
+      collateralToken: collateralToken.address,
+      oracle,
+      irm,
+      lltv,
+    };
+
+    const irmContract = Irm__factory.connect(irm, this.provider);
+
+    const borrowRateView = await irmContract.borrowRateView(marketParams, market, { blockTag: this.blockTag });
+    const borrowApy = (
+      Math.exp(new BigNumberJS(borrowRateView.toString()).div(1e18).times(SECONDS_PER_YEAR).toNumber()) - 1
```

Comment:
Wondering why we need to transform `borrowRateView` to `BigNumberJS` when `borrowRateView` is already a `BigNumber`?